src/assets/styles/index.css

```diff
@@ -10,6 +10,23 @@ h1, h2, h3, h4, h5, h6, p, span {
   margin: 0;
 }
 
+h2 {
+  font-size: 1.5rem;
+
+  @media (min-width: 1280px) {
+    font-size: 2rem;
+  }
+}
+
+p {
+  color: var(--color-text-secondary);
+  font-size: 1rem;
+
+  @media (min-width: 1280px) {
```

Comment:
Let's use CSS variables for breakpoints

Reply:
This syntax doesn't work:

` @media (min-width: var(--breakpoint))`

This would be great here, but it's a future feature: https://publishing-project.rivendellweb.net/new-and-upcoming-css-custom-media/

So what I'm trying to get is preprocessing with SASS, but that requires reworking the files a bit.

Reply:
I see, in this case let's keep the hardcoded values for now then. No need to complicate things